<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>:::ADBKP:::</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pwr_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vol_up_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume Up</string>
    </property>
   </widget>
   <widget class="Line" name="right_line">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>5</y>
      <width>3</width>
      <height>480</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="left_line">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>5</y>
      <width>3</width>
      <height>480</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="vol_dn_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>490</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="Line" name="bottom_line">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>485</y>
      <width>326</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="top_line">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>3</y>
      <width>326</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="unlock_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Unlock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lp_pwr_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>L.P. Power</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;

&lt;p align=&quot;center&quot;&gt;ADBKP&lt;/p&gt;
&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/files/android_logo.png&quot;/&gt;
&lt;p id=&quot;name&quot; align=&quot;center&quot; style=&quot;font-size: 12px; font-family: 'Comic Sans MS'&quot;&gt;Made by:&lt;br&gt;
Vineeth Raj&lt;br&gt;
&lt;i&gt;vineethraj49@gmail.com&lt;/i&gt;
&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
